renovate.json

```diff
@@ -2,5 +2,20 @@
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "extends": [
     "config:base"
+  ],
+  "packageRules": [
+    {
+      "matchPackagePrefixes": ["ch.qos.logback:"],
+      "groupName": "logback packages"
+    },
+    {
+      "matchPackageNames": ["org.springframework.boot"],
+      "matchUpdateTypes": ["major"],
+      "enabled": false
+    },
+    {
+      "matchPackageNames": ["eclipse-temurin"],
+      "enabled": false
```

Comment:
renovate [claims to be smart about docker version updates](https://docs.renovatebot.com/docker/#preservation-of-version-precision) but doesn't seem to work with `eclipse-temurin`, maybe because it doesn't even have a minor version(?) anyways, this seems fine to manually update if/when we want